tests/conftest.py

```diff
@@ -27,14 +30,42 @@ def manager():
             os.chdir(oldDirectory)
     return manager
 
-## Command-line option to skip very slow tests
 
 def pytest_addoption(parser):
+    # option to skip very slow tests
     parser.addoption('--fast', action='store_true', help='skip very slow tests')
+    # option to skip parser generation
+    parser.addoption('--skip-pegen', action='store_true', help='generate the parser before running tests')
```

Comment:
`help` message seems to be backwards.

Reply:
Fixed 7becaf085edf0291de565e42301ab0cb88706dc1